### PR TITLE
feat(all SCs): migrate all contracts to std 1.1 (main)

### PR DESCRIPTION
**Pull request by P-Yevhenii**

### CODE REVIEW

**Review by ShadoySV — COMMENTED** [review]
```Decimal256Ext::saturating_sub``` and ```DecimalCheckedOps::checked_add``` are not longer needed

File: contracts/tokenomics/xastro_token/src/contract.rs [hunk 1]
```diff
@@ -279,6 +279,7 @@ pub fn execute(
             marketing,
         } => execute_update_marketing(deps, env, info, project, description, marketing),
         ExecuteMsg::UploadLogo(logo) => execute_upload_logo(deps, env, info, logo),
+        _ => Err(ContractError::Unauthorized {}),
```
Comment (epanchee) [inline contracts/tokenomics/xastro_token/src/contract.rs:282]: Why? Without this line we receive more descriptive error in runtime. I suggest to remove this line.

File: packages/astroport/src/generator_proxy.rs [hunk 2]
```diff
@@ -66,24 +62,29 @@ pub enum CallbackMsg {
...
+    #[returns(Uint128)]
     Deposit {},
     /// Returns the amount of rewards to be distributed
+    #[returns(Uint128)]
     Reward {},
     /// Returns the amount of pending rewards which can be claimed right now
+    #[returns(Uint128)]
     PendingToken {},
     /// Returns the 3rd party reward token contract address
+    #[returns(Addr)]
```
Comment (ShadoySV) [inline packages/astroport/src/generator_proxy.rs:81]:
```suggestion
    #[returns(RewardInfoResponse)]
```

**Review by ShadoySV — CHANGES_REQUESTED** [review]

File: contracts/tokenomics/xastro_token/src/contract.rs (hunk 1 above)
Comment (ShadoySV) [reply contracts/tokenomics/xastro_token/src/contract.rs:282]: Another method was added and we have to ignore it

**Review by ShadoySV — APPROVED** [review]

**Review by epanchee — APPROVED** [review]

**Pull request merged**